.env.example

```
@@ -772,3 +772,24 @@ LANGFUSE_ENABLED=false
 # OTEL_SERVICE_NAME=neurolink-ai
+
+# =============================================================================
+# REAL-TIME VOICE SERVER (voice-server command)
+# =============================================================================
+
+# Cartesia TTS — https://cartesia.ai
+CARTESIA_API_KEY=
+# Optional: override if Cartesia provides a different WS endpoint for your account/region
+CARTESIA_WS_BASE_URL=wss://api.cartesia.ai/tts/websocket
+# Optional: API version pin used in Cartesia WS query params
+CARTESIA_API_VERSION=2025-04-16
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CARTESIA_API_VERSION key should go before the CARTESIA_WS_BASE_URL key

(UnorderedKey)